slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -146,27 +147,37 @@ public void importDataWithFilter(final InputStream input, final List<String> pub
   }
 
   @Override
-  public void export(final OutputStream output) {
+  public void exportData(final OutputStream output) {
     try {
       LOG.info("Exporting slashing protection database");
-      interchangeManager.export(output);
+      interchangeManager.exportData(output);
       LOG.info("Export complete");
     } catch (IOException e) {
       throw new RuntimeException("Failed to export database content", e);
     }
   }
 
   @Override
-  public void exportWithFilter(final OutputStream output, final List<String> pubkeys) {
+  public void exportDataWithFilter(final OutputStream output, final List<String> pubkeys) {
     try {
       LOG.info("Exporting slashing protection database for keys: " + String.join(",", pubkeys));
-      interchangeManager.exportWithFilter(output, pubkeys);
+      interchangeManager.exportDataWithFilter(output, pubkeys);
       LOG.info("Export complete");
     } catch (IOException e) {
       throw new RuntimeException("Failed to export database content", e);
     }
   }
 
+  @Override
+  public IncrementalExporter createIncrementalExporter(final OutputStream out) {
+    try {
```

Comment:
nit: Do we want a log here similar to exportData, e.g.
`LOG.info("Exporting slashing protection database");`
 or was it an intentional omission?

Reply:
Will need to move the logging though into the IncrementalExporter instead though